packages/wdio-runner/tests/reporter.test.js

```diff
@@ -159,7 +159,7 @@ describe('BaseReporter', () => {
             reporterSyncTimeout: 100
         })
 
-        setTimeout(() => (reporter.reporters[0].inSync = true), 110)
+        setTimeout(() => (reporter.reporters[0].inSync = true), 112)
```

Comment:
this test is often failing for me and even sometimes in CI.
Adding 1ms increases stability by 90%, had no failures with 2ms added